ambry-named-mysql/src/main/resources/NamedBlobsSchema.ddl

```diff
@@ -6,9 +6,11 @@ CREATE TABLE IF NOT EXISTS named_blobs_v2 (
     version bigint NOT NULL,
     blob_id varbinary(50) NOT NULL,
     deleted_ts datetime(6) DEFAULT NULL,
+    gg_modi_ts datetime(6) DEFAULT (CURRENT_TIMESTAMP(6) - INTERVAL 61 DAY),
```

Comment:
QQ, May I know why do we made this change? Will this affect the GG replication since it leverage this value to identify the replication priority. And can you please remind me why we don't have gg_modi_ts and gg_status in the schema before? And the "GG triggers" like what we have in "database/AMBRYBLOBS/schema/ddl/00-table.sql". I remember MYSQL team told me that we have to have this in schema to enable GG replication, but it looks like currently we don't have it but the GG replication are still been enabled.

Reply:
@SophieGuo410 you are right, we need this 'gg_modi_ts' field so GG replication can work. What we did is provided the table creation schema to sql team (and create a RB for internal source), then the sql team will apply that change. So the internal and external kind of going out sync for this schema. Even though it doesn't impact the deployed service. I am adding it back so they are in sync.

Reply:
I see, so in order to keep it in sync I guess we need the gg_status and the GG trigger condition as well right? Just like what we have in "database/AMBRYBLOBS/schema/ddl/00-table.sql"

Reply:
I will add the gg_status. The GG replication setting needs some work at mysql team side as well. So we can't copy the triggers over. Otherwise it will get error eg ```Unknown column 'gg_status' in 'NEW'```